src/test/java/com/diffmin/AppTest.java

```diff
@@ -178,10 +176,7 @@ private static void runTests(TestResources sources) throws Exception {
         File f1 = sources.prevPath.toFile();
         File f2 = sources.newPath.toFile();
         App app = new App();
```

Comment:
I don't want to initialise this here as  `Main.patchAndGenerateModel(f1, f2);` takes care of it but I had to print the model later so I had to initialise it to use `displayModifiedModel`.

But I think this will vanish when we separate methods like these into a separate utility package.

Reply:
Make `displayMOdifiedModel` static?